src/components/molecules/Card/component.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { FC } from 'react';
+import { Props } from './props';
+import styled from '@emotion/styled';
+
+export const Card = styled.div<Props>``;
```

Comment:
Delete defaultProps and write CSS properties in the body, by using condition. E.g: `background: #fff || ${ background }`